stories/fields/NumberInput.stories.tsx

```diff
@@ -39,13 +39,17 @@ export default {
 }
 
 export function _NumberInput(props: NumberInputProps) {
-  const [outputValue, setOutputValue] = useState<number | undefined | '∅'>('∅')
+  const [outputValue, setOutputValue] = useState<number | undefined>(props.value)
+
+  useEffect(() => {
+    setOutputValue(props.value)
+  }, [props.value])
 
   return (
     <>
-      <NumberInput {...props} onChange={setOutputValue} />
+      <NumberInput {...props} onChange={setOutputValue} value={outputValue} />
 
-      {outputValue !== '∅' && <Output value={outputValue} />}
+      <Output value={outputValue} />
```

Comment:
The output is there to show what is the value called by `onChange`. It is important for tests since we don't want an `undefined` that could validate a test that would be in fact wrong since `undefined` represents some real behaviors. `∅` is a hack to show that. We could even move this pattern into `Output`.

Reply:
Since we are now using controlled components, initial value of `outputValue` is passed to component, so it should be of the type expected by the component. For tests, may be you want to listen to `onChange` calls ?